src/components/Review/DecisionAreaNEW.tsx

```diff
@@ -0,0 +1,128 @@
+  const [updateReviewResponse] = useUpdateReviewResponseMutation()
+
+  useEffect(() => {
+    if (toggle != previousToggle) {
+      setReview(reviewResponse)
+      setIsOpen(true)
+      setPreviousToggle(toggle)
+    }
+  }, [toggle])
+
+  const submit = () => {
+    updateReviewResponse({
+      variables: {
+        id: review.id,
+        decision: review.decision,
+        comment: review.comment,
+      },
+    })
+    setIsOpen(false)
+  }
+
+  return (
+    <Modal
+      closeIcon
+      open={isOpen}
+      onClose={() => setIsOpen(false)}
+      size="fullscreen"
+      style={{ margin: 0, background: 'transparent' }}
+    >
+      {!isOpen ? null : (
+        <Segment
+          floated="right"
+          style={{
+            backgroundColor: 'white',
+            height: '100vh',
+            width: '300px',
+          }}
+        >
+          <Segment basic>
+            <Header>{strings.TITLE_DETAILS}</Header>
+            <SummaryViewWrapperNEW {...summaryViewProps} />
+          </Segment>
+          <Segment basic>
+            <Header as="h3">{strings.LABEL_REVIEW}</Header>
+            {/* // TODO: Change to list options dynamically? */}
+            <Radio
+              label={strings.LABEL_REVIEW_APPROVE}
+              value={strings.LABEL_REVIEW_APPROVE}
+              name="decisionGroup"
+              checked={review.decision === ReviewResponseDecision.Approve}
+              onChange={() =>
+                setReview({
+                  ...review,
+                  decision: ReviewResponseDecision.Approve,
+                })
+              }
+            />
+            <Radio
+              label={strings.LABEL_REVIEW_RESSUBMIT}
+              value={strings.LABEL_REVIEW_RESSUBMIT}
+              name="decisionGroup"
+              checked={review.decision === ReviewResponseDecision.Decline}
+              onChange={() =>
+                setReview({
+                  ...review,
+                  decision: ReviewResponseDecision.Decline,
+                })
+              }
+            />
+          </Segment>
+          <Segment basic>
+            <Header as="h3">{strings.LABEL_COMMENT}</Header>
+            <TextArea
+              rows={6}
+              style={{ width: '100%' }}
+              defaultValue={review.comment}
+              onChange={(_, { value }) => setReview({ ...review, comment: String(value) })}
+            />
+          </Segment>
+          <Segment basic>
+            <Button
+              color="blue"
+              basic
+              onClick={submit}
+              disabled={
+                !review.decision ||
+                (review.decision === ReviewResponseDecision.Decline && !review.comment)
+              }
+              content={strings.BUTTON_SUBMIT}
+            />
+            {review.decision === ReviewResponseDecision.Decline && !review.comment && (
+              <p style={{ color: 'red' }}>{messages.REVIEW_RESUBMIT_COMMENT}</p>
+            )}
```

Comment:
I saw this change here. Looks good : )
<img width="298" alt="Screen Shot 2021-03-06 at 5 15 54 PM" src="https://user-images.githubusercontent.com/16461988/110194895-eb72c800-7e9f-11eb-8796-267c250072a7.png">